tabs/osd.js

```diff
@@ -370,7 +388,30 @@ OSD.constants = {
 OSD.chooseFields = function () {
   var F = OSD.constants.ALL_DISPLAY_FIELDS;
   // version 3.0.1
-  if (semver.gte(CONFIG.flightControllerVersion, "3.0.1")) {
+  if (semver.gte(CONFIG.flightControllerVersion, "3.1.0")) {
+    OSD.constants.DISPLAY_FIELDS = [
+      F.RSSI_VALUE,
+      F.MAIN_BATT_VOLTAGE,
+      F.CROSSHAIRS,
+      F.ARTIFICIAL_HORIZON,
+      F.HORIZON_SIDEBARS,
+      F.ONTIME,
+      F.FLYTIME,
+      F.FLYMODE,
+      F.CRAFT_NAME,
+      F.THROTTLE_POSITION,
+      F.VTX_CHANNEL,
+      F.CURRENT_DRAW,
+      F.MAH_DRAWN,
+      F.GPS_SPEED,
+      F.GPS_SATS,
+      F.ALTITUDE,
+      F.PID_ROLL,
```

Comment:
It would be tidier to assign the values to `OSD.constants.DISPLAY_FIELDS` for >= 3.0.1, and then just add the 3 new elements for >= 3.1.0, instead of duplicating the whole list.